### PR TITLE
dag pausing should pause queued tasks as well

### PR DESCRIPTION
**Pull request by mistercrunch**

### CODE REVIEW

File: airflow/jobs.py [hunk 1]
```diff
@@ -570,8 +570,8 @@ def prioritize_queued(self, session, executor, dagbag):
                     self.logger.info("Pickling DAG {}".format(dag))
                     pickle_id = dag.pickle(session).id
 
-                if dag.dag_id in overloaded_dags or dag.concurrency_reached:
-                    overloaded_dags.add(dag.dag_id)
+                if dag.dag_id in dag_blacklist or dag.concurrency_reached:
```
Comment (aoen) [inline airflow/jobs.py]: Don't need the first part of the if since they are already in the blacklist

Comment (mistercrunch) [inline airflow/jobs.py]: that's for the `or concurrency_reached`, I can break it down as an elif, though `dag_blacklist` is a set so it won't grow anyways

**Comment by aoen** [comment]
LGTM

**Comment by mistercrunch** [comment]
The previous setup would allow to queue to empty out even when the DAG was paused. No new tasks would get queued, but the ones in the queue were allowed to drain...

**Pull request merged**